<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wakaranai</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>711</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ходы партии и анализ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Следующий ход первого игрока</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>601</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>601</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1343</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Движок</string>
    </property>
    <addaction name="actionConnectEngine"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>БД</string>
    </property>
    <addaction name="actionConnectDB"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionSetGame"/>
    <addaction name="actionSaveGame"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionContinueGame"/>
    <addaction name="separator"/>
    <addaction name="actionCopyGame"/>
    <addaction name="actionPasteGame"/>
    <addaction name="separator"/>
    <addaction name="actionChangePlayerName"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAboutProgram"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPasteGame">
   <property name="text">
    <string>Вставить игру (.kif)</string>
   </property>
  </action>
  <action name="actionConnectEngine">
   <property name="text">
    <string>Подключение</string>
   </property>
  </action>
  <action name="actionConnectDB">
   <property name="text">
    <string>Подключение</string>
   </property>
  </action>
  <action name="actionAboutProgram">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="text">
    <string>Начать новую игру</string>
   </property>
  </action>
  <action name="actionSetGame">
   <property name="text">
    <string>Загрузить игру</string>
   </property>
  </action>
  <action name="actionSaveGame">
   <property name="text">
    <string>Сохранить игру</string>
   </property>
  </action>
  <action name="actionCopyGame">
   <property name="text">
    <string>Копировать игру</string>
   </property>
  </action>
  <action name="actionChangePlayerName">
   <property name="text">
    <string>Изменить имя игрока</string>
   </property>
  </action>
  <action name="actionContinueGame">
   <property name="text">
    <string>Продолжить с текущей позиции</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
